packages/orbit-components/src/primitives/TooltipPrimitive/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -1,37 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Yes, this snapshot was a kind of one-of-a-kind test in Orbit. It was not even asserting that the tooltip rendered, because it was only rendering the target `<p>`… It was asserting that the styles from styled components were correctly computed, I guess…
To be honest, I didn't find any value in this test, and the snapshot failed (expectedly) upon migrating to Tailwind. Therefore, I decided to remove this and add a basic (but missing) test that asserts that the tooltip appears on hover of its target.